CarRental/CommonLibrary/Enums/Gender.cs

```diff
@@ -1,12 +1,18 @@
 ﻿using Newtonsoft.Json.Converters;
+using System.Runtime.Serialization;
 using System.Text.Json.Serialization;
 
 namespace CommonLibrary.Enums;
 
 [JsonConverter(typeof(StringEnumConverter))]
 public enum Gender
 {
-    M,
-    F,
-    O,
+    [EnumMember(Value = "M")]
```

Comment:
czemu to tak przerobiles

Reply:
bo to pozwala zmapować do czytelnego enuma, gdzie mamy pełne nazwy Male, Female...